docs/proposals/automated-node-removal.md

```diff
@@ -0,0 +1,60 @@
+# Proposal: Automation of node removal procedure
+
+Last updated: 24.06.2021
+
+## Abstract
+
+Implementation of the option to delete CSI CRs from a node, which going to be removed. 
+
+## Background
+
+User manual actions for node removal are described in [doc](https://github.com/dell/csi-baremetal/blob/master/docs/node-removal.md). The goal of this POC is automating most of the steps of that procedure. 
```

Comment:
This is not POC, but proposal for feature implementation.

Reply:
fixed